### DISCUSSION TITLE
IconLast and IconFirst missing in the DS

### DISCUSSION THREAD

**Discussion by LauGroupie95** [buildo/bento-design-system#344] [General]
Hi @gabro and @veej ,
I'm trying to importa the IconLast and the IconFirst in my project, but It seems they're not present in the DS (or maybe I'm not able to find them). Please, can you help me?

Here's an example of what I'm talking about:
![image](https://user-images.githubusercontent.com/63547770/183041701-ab6dbad4-69d0-4266-8178-7ce87e9e14df.png)
![image](https://user-images.githubusercontent.com/63547770/183041792-8c24946e-be8f-4b03-b875-4845a797e343.png)

**Comment by veej** [comment]
Hi @LauGroupie95,
those icons are not standard from Bento, so they must be implemented in your project (Agnese forgot to mark them as such, but now you should see them marked correctly in Figma)
<img width="229" alt="image" src="https://user-images.githubusercontent.com/925635/183046938-b6bfcea2-1a27-4eb9-8dc9-c36a3d4bf24f.png">

**Reply by gabro** [reply]
I've converted this to a discussion since it's not something we will implement in Bento.

That said, you can implement them in your project using the `svgIconProps` utility. You can take a look at any icon in Bento, for example https://github.com/buildo/bento-design-system/blob/main/packages/bento-design-system/src/Icons/IconEye.tsx, and copy the approach.

**Reply by LauGroupie95** [reply]
Ok, thank you everybody, now it's marked with the "+" as it's shown in the image :)
